prisma/prisma-types.ts

```diff
@@ -391,7 +391,7 @@ export const prismaPoolBatchSwapWithSwaps = Prisma.validator<Prisma.PrismaPoolBa
 
 export type PrismaPoolBatchSwapWithSwaps = Prisma.PrismaPoolBatchSwapGetPayload<typeof prismaPoolBatchSwapWithSwaps>;
 
-export const prismaPoolWithDynamic = Prisma.validator<Prisma.PrismaPoolArgs>()({
+export const prismaPoolWithDynamicAndHook = Prisma.validator<Prisma.PrismaPoolArgs>()({
```

Comment:
I'd say `prismaPoolAndHookWithDynamic` follows the existing pattern a bit better.
@franzns - since this is more of a backend/db thing than an SOR thing, do you have a preference?

Reply:
No strong opinion